### DISCUSSION TITLE
Best services/methods to correctly deploy the template? (AWS, Supabase, etc)

### DISCUSSION THREAD

**Discussion by josephna76** [fastapi/full-stack-fastapi-template#1169] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

I ran the project locally with the following and got both the websocket and http routes to work.

### Description

I'm looking for thoughts on the best way to deploy this dockerized template (which includes websocket routes). Was there a deployment service/method that @tiangolo had in mind when creating this template?

For my example, fastapi is the "backend" api, which serves separate ios/android/web apps. Along with the typical https (post, get) routes, I also have websocket routes (wss). I was able to get the websocket route to work properly on my local computer/ios simulator with the configuration above. 

I was originally going to run the app on aws lambdas, with the database separated out into a separate service, but read online that lambdas doesn't support wss out of the box and requires aws api gateway. 

Another method I was considering was to use a service like supabase, which apparently also supports streaming in beta. 

tl;dr: I'm nervous to architecture my own solution given the features already in place in the template and want to leverage said features in the best possible way when deploying.

### Operating System

macOS

### Operating System Details

Intel chip, not ARM.

### Python Version

Python Version 3.12

**Comment by Happily-Coding** [comment]
Hey there, you might want to see:
https://github.com/tiangolo/full-stack-fastapi-template/blob/master/deployment.md
https://github.com/tiangolo/full-stack-fastapi-template/issues/322

**Reply by fanpeng-kong** [reply]
@Happily-Coding  have you been able to deploy the stack following the deployment instructions?  I was able to deploy the stack in my Cloud VM and can access the 'Traefik' and 'Adminer' through my sub-domains. However, I had "Your connection is not private" issue when I try to access the frontend and backedn API docs and can't proceed. The "Traefik" dashboard showed all these services running normally. 

Just in case you have some successful stories...

**Reply by realsbd** [reply]
I am currently having problem to deploy to my Digital Ocean VPS, I have followed the guide and when I made a PR in my repo, I got this error in Action runner
![image](https://github.com/user-attachments/assets/941b8044-feed-460d-94a7-c9fa2bca6e16)
